packages/headless/src/controllers/search-parameter-manager/headless-search-parameter-manager.ssr.test.ts

```diff
@@ -0,0 +1,50 @@
+import {SSRSearchEngine} from '../../app/search-engine/search-engine.ssr';
+import {ControllerDefinitionWithProps} from '../../app/ssr-engine/types/common';
+import {buildMockSSRSearchEngine} from '../../test/mock-engine';
+import {
+  SearchParameterManager,
+  buildSearchParameterManager,
+} from './headless-search-parameter-manager';
+import {
+  SearchParameterManagerBuildProps,
+  defineSearchParameterManager,
+} from './headless-search-parameter-manager.ssr';
+
+jest.mock('./headless-search-parameter-manager');
+const buildSearchParameterManagerMock = jest.mocked(
+  buildSearchParameterManager
+);
+
+type SearchParameterManagerDefinitionType = ControllerDefinitionWithProps<
+  SSRSearchEngine,
+  SearchParameterManager,
+  SearchParameterManagerBuildProps
+>;
+describe('define search parameter manager', () => {
```

Comment:
prettier ?

Reply:
unfortunately prettier doesn't look at spacing like this.